internal/receiver/discoveryreceiver/endpoint_tracker_test.go

```diff
@@ -721,8 +721,10 @@ func TestEntityEmittingLifecycle(t *testing.T) {
 	}, 1*time.Second, 50*time.Millisecond)
 
 	// Ensure that entities are not emitted anymore
-	time.Sleep(60 * time.Millisecond)
```

Comment:
We need to ensure that there are no new events are sent after https://github.com/signalfx/splunk-otel-collector/pull/5199/files#diff-28129c53a68d9dbe21062b3d5f8a169a87e9addf66b86f94631237cb63b85cf6R719.

We cannot check right away because this can happen before the 719 line.

You can either increase the sleep timeout here or add a synchronization so this line is blocked until 719.

Reply:
Add `<-waitChan` here and `close(waitChan)` after 719